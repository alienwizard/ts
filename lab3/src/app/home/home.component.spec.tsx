import * as React from "react";
import Home from "./home.component";
import * as renderer from "react-test-renderer";

describe('home', function(){
    beforeEach(function() {

    })
    it('renders correctly', () => {
        const tree = renderer.create(
            <Home />
        ).toJSON();
        expect(tree).toMatchSnapshot();
    })
})