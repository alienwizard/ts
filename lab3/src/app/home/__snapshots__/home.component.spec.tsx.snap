// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`home renders correctly 1`] = `
<div
  className="llduHn"
>
  <h1>
    Welcome!
  </h1>
</div>
`;
